scalastyle-config.xml

```diff
@@ -42,7 +42,7 @@
             <parameter name="maxParameters"><![CDATA[8]]></parameter>
         </parameters>
     </check>
-    <check level="warning" class="org.scalastyle.scalariform.MagicNumberChecker" enabled="true">
+    <check level="warning" class="org.scalastyle.scalariform.MagicNumberChecker" enabled="false">
```

Comment:
disabled this rule for now as its limited to 100 warnings and all 100 are magic numbers, need to cleanup some before enabling it back